eclipse/plugin.xml

```diff
@@ -1039,11 +1039,14 @@
                   <reference
                         definitionId="saros.ui.definitions.isSarosSessionRunning">
                   </reference>
+                  <reference
+                        definitionId="saros.ui.definitions.isSarosSessionHost">
+                  </reference>
                   <reference
                         definitionId="saros.ui.definitions.contactsSelected">
                   </reference>
                   <reference
-                        definitionId="saros.ui.definitions.isSarosSessionHost">
+                        definitionId="saros.ui.definitions.contactsOnline">
```

Comment:
I am really not familiar with the Eclipse plugin.xml config stuff. Could you explain what these changes do?

You add all of this config stuff which makes it look like the online check should be done by eclipse, but then call the tester manually (see comment in `SarosView`). If it wasn't added for that purpose, why was it added?

Reply:
Seems the author of this incredible stuff has left the building long ago. Maybe you talk to @m273d15 . I can tell you his answer. It should be the same as mine: WTF is this ******* ....

It is really complicated, it is a waste of time, coding this stuff takes minutes, define it as XML takes hours with frustrating trail and errors.

Reply:
Basically it is: Add the command (menu item) "Add contacts to session" to the any context menu if the following conditions holds true.

You are connected, a session is running, you are the host, the current selection(s) are contacts and these contacts are currently online.